permissions/plugin-atlassian-jira-software-cloud.yml

```diff
@@ -7,6 +7,5 @@ paths:
   - "com/atlassian/jira/software/cloud/jenkins/atlassian-jira-software-cloud"
 developers:
   - "lenin_atlassian"
-  - "vikrambollu_atlassian"
   - "bgvozdev"
-  - "nsurendran"
+  - "thombergs"
```

Comment:
This is my GitHub username as well as my Jenkins LDAP user name.

The two people I removed are no longer with our team here at Atlassian and will no longer work on the plugin